symlinks.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+CONFIG_DIR=/config
+CONFIG_FILES=$CONFIG_DIR/*
+
+if [ -d "$CONFIG_DIR" ]; then
+  echo "creating symlinks for files in: $CONFIG_DIR"
+  for f in $CONFIG_FILES
+  do
+    ln -sf $f /app/$(basename $f)
+    echo $f /app/$(basename $f)
+  done
+else
+  echo "$CONFIG_DIR does not exist, symlinks were not created."
+fi
+echo "running startup.sh..."
+/app/startup.sh
```

Comment:
The should probably be `exec /app/startup.sh` so that the process id gets transfered